buildenv/docker/mkdocker.sh

```diff
@@ -581,6 +591,12 @@ bootjdk_url() {
 
 install_bootjdks() {
   local versions="8 11 16"
+if [ $jdk_to_build != all ] ; then
+   versions=$jdk_to_build
+fi
+if [ $jdk_to_build = 17 ] ; then
+   versions="16"
+fi
```

Comment:
Please follow the existing indentation pattern (two spaces per level).

I suggest that one should support building more than one jdk, via, for example, ​`--jdk='11 17'`:
```
  if [ "$jdk_to_build" != all ] ; then
    versions="${jdk_to_build/17/16}"
  fi
```